python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -199,6 +203,16 @@ def partition_set_name(self) -> Optional[str]:
 
         return self.partition_set_origin.partition_set_name
 
+    @property
+    def job_name(self) -> Optional[str]:
```

Comment:
I also considered  making `job_name` an attr of the named tuple that the user sets. But then you have to do checking in the constructor to make sure the passed job name lines up with what we expect (ie matches the partition set name for job backfill and is `__ASSET_JOB` for asset backfills). it seemed like extra work on the caller of `PartitionBackfill` for little benefit